test/e2e-encryption/encryption_test.go

```diff
@@ -1,7 +1,23 @@
 package e2e_encryption
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	test "github.com/openshift/cluster-kube-apiserver-operator/test/library"
+)
 
 func TestEncryptionKeyEtcd(t *testing.T) {
-	t.Log("add tests here")
+	kubeConfig, err := test.NewClientConfigForTest()
+	require.NoError(t, err)
```

Comment:
why not simply use std library?

Reply:
Just following the pattern of the existing tests.

Reply:
ok, ic.